src/Campaigns/Actions/LoadCampaignDetailsAssets.php

```diff
@@ -14,30 +12,33 @@ class LoadCampaignDetailsAssets
     /**
      * @unreleased
      */
-    public function __invoke(Campaign $campaign)
+    public function __invoke()
```

Comment:
The `CampaignsAdminPage.php` file has a call for this file where the `$campaign` is passed as a parameter, make sure to remove it from there as it's no longer necessary.